templates/base.html

```diff
@@ -30,24 +30,24 @@
     sizes="16x16">
   <link rel="mask-icon" href="https://www.voteamerica.com/img/safari-pinned-tab.svg"
     color="#ff4400">
-  <meta name="description" content="Track your Georgia mail-in ballot">
-  <meta name="og:description" content="Track your Georgia mail-in ballot">
+  <meta name="description" content="Help make sure every voice in Georgia is heard">
+  <meta name="og:description" content="Help make sure every voice in Georgia is heard">
   <meta name="theme-color" content="#fff">
   <meta property="og:type" content="website">
-  <meta property="og:title" content="gaballot.org - Track your Georgia mail-in ballot">
+  <meta property="og:title" content="gaballot.org - Help make sure every voice in Georgia is heard">
   <meta property="fb:app_id" content="218198872691029">
   <meta name="twitter:card" content="summary_large_image">
   <meta name="twitter:site" content="@VoteAmerica">
   <meta property="og:image"
-    content="https://s3.amazonaws.com/appforest_uf/f1605654010173x452403538515414600/turnout.png">
+    content="https://www.voteamerica.com/share/v3/generic.png">
 
   <!-- Bootstrap CSS -->
   <link rel="stylesheet"
     href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
     integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
     crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <link href="/static/styles.css?1" rel="stylesheet">
+  <link href="/static/styles.css" rel="stylesheet">
```

Comment:
as noted in other review, i actually do think this might be necessary to work around the cloudflare cache. you might also want to "bump" the version for the /static/homepage.js include as well.